fuse-ts.c

```diff
@@ -33,11 +33,14 @@ char *start_time = NULL;
 char *prefix = NULL;
 int inframe = 0;
 int outframe = 0;
+int frames_per_second = 50;
```

Comment:
This must be 25 for several reasons.

Reply:
OK. We can take 25 as default.

Reply:
YOU did choose 25 as the default ;)

See 890789da85031f6c14080f45aeb0a3e32e48f871

Reply:
I see, then it was a "real" miss teak.